AstroEQ-Firmware/AstroEQ/AstroEQ.c

```diff
@@ -1448,6 +1448,13 @@ void gotoMode(byte axis){
     motorStart(axis); //Begin PWM
 }
 
+void clearGotoMode(const byte axis){
+	if (motionIsGoto(cmd.GVal[axis])){
```

Comment:
This should work because :G will only change GVal if the motors are already stopped.

Reply:
Makes sense.